### PR TITLE
move2

### PR DESCRIPTION
**Pull request by cleemansen**

### CODE REVIEW

**Comment by cleemansen** [comment]
Hello @andreakoelzsch , can you please provide a solution to access the `timestamps` in the `move2` object.

I'm facing the following error executing the sample sdk now with `move2`:

```
> source("~/workspace/Template_R_Function_App/sdk.R")
[DEBUG] parse stored configuration: './app-configuration.json'
[INFO] app will be started with configuration:
{
  "sdk": "MoveApps R SDK",
  "year": 2006
}

Attaching package: ‘lubridate’

The following objects are masked from ‘package:base’:

    date, intersect, setdiff, union

[DEBUG] Read input of type 'move::moveStack'
[DEBUG] sourceFile: ./data/raw/input3.rds
[DEBUG] Reading RDS from file './data/raw/input3.rds'
[INFO] Welcome to the MoveApps R SDK
[1] "ERROR:  Error in data@timestamps: no applicable method for `@` applied to an object of class \"move2\"\n"
[DEBUG] errorFile: ./data/output/error.log
[DEBUG] Writing to file ./data/output/error.log
Error in data@timestamps :
no applicable method for `@` applied to an object of class "move2"
```

This should be the affected code snippet (works w/ `move1`, not w/ `move2` anymore)

```
result <- if (any(lubridate::year(data@timestamps) == year)) { 
    data[lubridate::year(data@timestamps) == year]
  }
```

Thanks

**Comment by andreakoelzsch** [comment]
> 

Hi Clemens, I have changed this to move2 notation, is working fine locally. Will check now your next PR

**Review by andreakoelzsch — APPROVED** [review]

**Comment by cleemansen** [comment]
> I have changed this to move2 notation, is working fine locally

Thanks! Please don't forget to push your adjustments to this PR - I'm very interested in the new notation. 
And also please keep an eye on the existing Unit Tests whether they are still green.

File: CHANGELOG_SDK.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 # Changelog SDK
 
-## 2023-06-20 `v3.1.0`
+## 2023-06-22 `v3.1.0`
```
Comment (cleemansen) [inline CHANGELOG_SDK.md:3]: the idea is to omit the concrete day and just mention year and month. IMHO the day is pretty useless and is not practical to guess the day of a future merge.

Comment (andreakoelzsch) [reply CHANGELOG_SDK.md:3]: ok, then there will be many same months now, but ok. I am working on the test files now...

Comment (andreakoelzsch) [reply CHANGELOG_SDK.md:3]: will adapt the cahngelog directly on main

**Comment by cleemansen** [comment]
Complete diff of the `move2` introduction: https://github.com/movestore/Template_R_Function_App/compare/c4d6424..905c5bb

**Pull request merged**